src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="app">
-    <Login msg="Welcome to Your Vue.js App"/>
```

Comment:
You can write components on one line for a cleaner look and legibility.
`<Login v-if='whatever'/>`